packages/meter/README.md

```diff
@@ -43,3 +43,7 @@ import { Meter } from '@swc-uxp-wrappers/meter';
 ```html
 <sp-meter></sp-meter>
 ```
+
+## Known Issues
+- Duplicate labels appear in React context
```

Comment:
It's not an issue just in React Context, but in vanilla and even on Web as well. 
Mention the workaround, to use label attribute to provide contextual name to the meter.

Reply:
Added the workaround